### DISCUSSION TITLE
Not showing error/suggestion when not imported

### DISCUSSION THREAD

**Discussion by shoaibsharif** [vuejs/language-tools#1270] [Q&A] [answered]
I am not getting warning while I am trying to do the following:

![CleanShot 2022-05-14 at 18 58 55](https://user-images.githubusercontent.com/29075110/168451995-3f488fc1-a4e9-4805-8c60-43514542997a.png)

but in svelte extension, they are working as expected. For example, if I do the following example below:

It will shows me error with red squiggly line and on hover it tells me correct error:

![CleanShot 2022-05-06 at 14 33 26](https://user-images.githubusercontent.com/29075110/167205571-7f6c59b1-79cc-474a-bdd2-77bc994c1bd3.png)

**Comment by johnsoncodehk** [comment, marked as answer]
Hi @shoaibsharif, please make sure you have config `"checkJs": true` in tsconfig / jsconfig because you're using javascript.
